### PR TITLE
remove echo statements in tests (part 1)

### PR DESCRIPTION
**Pull request by ringabout**
Too many echo statements in the results of megatests, which is hard to debug and reason with.

It helps https://github.com/nim-lang/Nim/pull/19972

Imo lots of echo are unnecessary and has some substitutes:

- echo "ok" at the end of the file is unnecessary
- echo can be replaced with doAssert in many cases

### CODE REVIEW

File: tests/vm/triangle_array.nim [hunk 1]
```diff
@@ -14,4 +10,4 @@ proc initCombinations: array[11, array[11, int]] =
   result[6][6 .. 10] =             [52,53,54,55,56]
 
 const combinations = initCombinations()
-echo combinations[6][10]
+doAssert $combinations[6][10] == "56"
```
Comment (ringabout) [inline tests/vm/triangle_array.nim]:
```suggestion
doAssert combinations[6][10] == 56
```

**Comment by ringabout** [comment]
This PRs focus on some simple cases where echos are at the end of the tests in most cases.

File: tests/vm/tyaytypedesc.nim [hunk 2]
```diff
@@ -18,4 +14,4 @@ const tokenTypeToNodeType = {
   ttWhitespace: ntWhitespace,
 }.enumTable(array[ttWhitespace..ttWhitespace, NodeType])
 
-echo tokenTypeToNodeType[ttWhitespace]
+doAssert $tokenTypeToNodeType[ttWhitespace] == "ntWhitespace"
```
Comment (konsumlamm) [inline tests/vm/tyaytypedesc.nim]:
```suggestion
doAssert tokenTypeToNodeType[ttWhitespace] == ntWhitespace
```

Comment (ringabout) [reply tests/vm/tyaytypedesc.nim]: Indeed, thanks you! According to https://github.com/nim-lang/Nim/issues/3357, it is unnecessary.

**Pull request merged**